pages/settings/teams.tsx

```diff
@@ -86,14 +97,14 @@ export default function Teams(props) {
             </div>
             <div>
               {!!teams.length &&
-                <TeamList teams={teams} onChange={loadTeams}>
+                <TeamList teams={teams} onChange={loadData}>
                 </TeamList>
               }
 
               {!!invites.length && <div>
                 <h2 className="text-lg leading-6 font-medium text-gray-900">Open Invitations</h2>
                 <ul className="border px-2 rounded mt-2 mb-2 divide-y divide-gray-200">
-                  {invites.map((team) => <TeamListItem onChange={loadTeams} key={team.id} team={team}></TeamListItem>)}
+                  {invites.map((team) => <TeamListItem onChange={loadData} key={team.id} team={team}></TeamListItem>)}
```

Comment:
Codacy has a fix for the issue: [Replace `{invites.map((team)·=>·<TeamListItem·onChange={loadData}·key={team.id}·team={team}></TeamListItem>` with `··{invites.map((team)·=>·(⏎······················<TeamListItem·onChange={loadData}·key={team.id}·team={team}></TeamListItem>⏎····················)`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689300)
```suggestion
                    {invites.map((team) => (
```